WebClient/src/communicator.ts

```diff
@@ -0,0 +1,35 @@
+﻿import * as signalR from "@microsoft/signalr";
+
+export class Communicator {
+    connection: any;
+
+    constructor() {
+        this.connection = new signalR.HubConnectionBuilder().withUrl("https://localhost:5001/testhub").build();
+    }
+
+    startConnection() {
```

Comment:
As discussed, this should not be exposed to the clients, rather do this when the Communicator object is instatiated.